tooling/kube_tool/other_params.rb

```diff
@@ -14,17 +14,19 @@ def OtherParams.create(os, version, container_runtime, cni_provider, bootstrap_c
       kubernetes_package_version = version
     end
 
+    cni_cluster_cidr = false
```

Comment:
These values need to be empty for Weave. When using weave this cause the Puppet run to fail. 

```
==> kube-master: Error: Evaluation Error: Error while evaluating a Resource Statement, Class[Kubernetes]: parameter 'cni_cluster_cidr' expects a value of type Undef or String, got Boolean (file: /tmp/modules/role/manifests/kubernetes/master.pp, line: 3, column: 3) on node kube-master.lan
```

Reply:
@mrwulf This PR looks good, I have tested it across multiple OS and network providers. I just need the above issue fixed for Weave and it will be ready to merge.

Reply:
Updated-